test/FESpacesTests/CurlConformingFESpacesTests.jl

```diff
@@ -43,6 +43,30 @@ dΩ = Measure(Ω,order)
 el2 = sqrt(sum( ∫( e⋅e )*dΩ ))
 @test el2 < 1.0e-10
 
+order = 1
```

Comment:
We have quite a lot of code replication here. Why dont we wrap a single test in a function provided the model and order and call it several times?

Reply:
We can do this in another PR, there are more tests like this in the file, and it is just a test, no src code